ansible/roles/baselayout/tasks/partials/repo/ubuntu1404.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+#
+# add PPA for gcc 4.9
+#
+
+
+- name: "repo : add Ubuntu Toolchain PPA"
+  apt_repository:
+    repo: 'ppa:ubuntu-toolchain-r/test'
+    state: present
+    update_cache: yes
+  register: has_updated_package_repo
```

Comment:
Maybe rename to `has_updated_toolchain_repo`

Reply:
It follows the variable name in other files under `repo`.